sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies.py

```diff
@@ -647,9 +648,10 @@ def get_backoff_time(self, settings):
 class StorageBearerTokenCredentialPolicy(BearerTokenCredentialPolicy):
     """ Custom Bearer token credential policy for following Storage Bearer challenges """
 
-    def __init__(self, credential, **kwargs):
+    def __init__(self, credential, *, audience: "BlobAudience", **kwargs):
         # type: (TokenCredential, **Any) -> None
-        super(StorageBearerTokenCredentialPolicy, self).__init__(credential, STORAGE_OAUTH_SCOPE, **kwargs)
+        audience = kwargs.pop('audience', STORAGE_OAUTH_SCOPE)
+        super(StorageBearerTokenCredentialPolicy, self).__init__(credential, str(audience), **kwargs)
```

Comment:
So if we just passed the value here, then we eliminate the necessity  of `str(audience)` in the super call. But it _should_ work in its current state as calling `str()` on a string has no effect, and in the case of a `BlobAudience` I've defined the `__str__` method to spit out the value.

Reply:
Regardless of what we decide for when to parse the string out, this class should definitely just take a string.